src/components/menu-right/DotAppearanceModal.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="modal-card" data-test="dot-appearance-modal">
+    <header class="modal-card-head">
+      <p class="modal-card-title">Dot type {{ dotTypeIndex }} Appearance</p>
+    </header>
+
+    <section class="modal-card-body">
+      <svg viewBox="-1 -1 2 2" class="column">
+        <Dot
+          :key="`menu-right-dot-${dotTypeIndex}-preview`"
+          :dotTypeIndex="dotTypeIndex"
+          :labeled="false"
+        />
+      </svg>
+      <div class="column">
+        <b-field>
+          <b-checkbox v-model="dotAppearance.filled">Filled</b-checkbox>
+        </b-field>
+        <b-field>
+          <b-checkbox v-model="dotAppearance.fwSlash"
+            >Forward-slashed</b-checkbox
+          >
+        </b-field>
+        <b-field>
+          <b-checkbox v-model="dotAppearance.bwSlash"
+            >Backward-slashed</b-checkbox
+          >
+        </b-field>
+        <b-field label="Fill Color">
+          <b-input v-model="dotAppearance.fill"></b-input>
+        </b-field>
+        <b-field label="Line Color">
+          <b-input v-model="dotAppearance.color"></b-input>
```

Comment:
For these color choices, I ended up using just a generic text input, Buefy/vue don't have a native color picker so this seemed easiest. This also allows users to input hex codes if they wanted.